docs/sequence/error_scenarios/PISPLinkingGrantConsentPhase1.puml

```diff
@@ -0,0 +1,204 @@
+@startuml
```

Comment:
I would make this sequence diagram shorter cutting off everything above `AUTHENTICATION-20`, and give a short textual summary.